app/web_hub_test.go

```diff
@@ -338,24 +338,37 @@ func TestHubConnIndexInactive(t *testing.T) {
 }
 
 func TestReliableWebSocketSend(t *testing.T) {
-	th := Setup(t)
-	defer th.TearDown()
-
 	testCluster := &testlib.FakeClusterInterface{}
-	th.Server.Cluster = testCluster
 
-	ev := model.NewWebSocketEvent("test_reliable_event", "", "", "", nil)
+	th := SetupWithClusterMock(t, testCluster)
```

Comment:
I didn't take a deep look into the race, but was the issue with resetting the cluster _after_ the testhelper was initialized?